src/compiler/compile/render_ssr/index.ts

```diff
@@ -160,6 +160,8 @@ export default function ssr(
 			${blocks}
 		});
 	`;
+
+	return [js, css];
```

Comment:
I'm not sure of the general code style across the Svelte project, but is it better to return:

``` { js, css } ```

rather than mystery-meat tuples:

``` [ js, css ] ```

It would make reasoning about the code simpler, further down the line.